Place WPT test in the right path

AlexsanderDamaceno opened the pull request:
#### c5b7d05b6cf367b5993454bc0ae483bf4ccf3aca
<pre>
Place WPT test in the right path
<a href="https://bugs.webkit.org/show_bug.cgi?id=290233">https://bugs.webkit.org/show_bug.cgi?id=290233</a>

Reviewed by Tim Nguyen.

Fix wrong location for WPT test block-in-inline-client-rects-001.html and block-in-inline-client-rects-001-expected.txt that
I imported in wrong path before.

* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/normal-flow/block-in-inline-client-rects-001-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/CSS2/block-in-inline-client-rects-001-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/normal-flow/block-in-inline-client-rects-001.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/CSS2/block-in-inline-client-rects-001.html.
* LayoutTests/imported/w3c/resources/import-expectations.json:

Canonical link: <a href="https://commits.webkit.org/292552@main">https://commits.webkit.org/292552@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/dd51c576dbdd3abea450a7a3d2f9d85bb959fe1b

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/96414 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/16028 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/6041 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/101482 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/46933 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/16324 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/24461 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/73491 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/30722 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/99417 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/12261 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/87128 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/53828 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/133/builds/12017 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/135/builds/4898 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/46261 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/82126 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/4994 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/103509 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/23481 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/17106 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/82538 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/23732 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/83154 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/81911 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/20550 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/26553 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/4015 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/16906 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/23444 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/23103 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/26583 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/24844 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Looks good, while you're there, can you add a line with `web-platform-tests/css/CSS2/normal-flow` here: https://searchfox.org/wubkat/rev/22cf494d8b679d4778bb90dbbf8a3d1f8747fa52/LayoutTests/imported/w3c/resources/import-expectations.json#55 ?

AlexsanderDamaceno committed:
Place WPT test in the right path
https://bugs.webkit.org/show_bug.cgi?id=290233

Reviewed by Tim Nguyen.

Fix wrong location for WPT test block-in-inline-client-rects-001.html and block-in-inline-client-rects-001-expected.txt that
I imported in wrong path before.

* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/normal-flow/block-in-inline-client-rects-001-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/CSS2/block-in-inline-client-rects-001-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/normal-flow/block-in-inline-client-rects-001.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/CSS2/block-in-inline-client-rects-001.html.
* LayoutTests/imported/w3c/resources/import-expectations.json:

Canonical link: https://commits.webkit.org/292552@main
